wgpu-core/src/instance.rs

```diff
@@ -162,46 +162,39 @@ impl<B: hal::Backend> Adapter<B> {
 
         let adapter_limits = raw.physical_device.limits();
 
-        let default_limits = wgt::Limits::default();
+        let mut limits = wgt::Limits::default();
```

Comment:
A disadvantage is that we cannot explicitly initialize the fields of this structure in `wgpu-core`, since it's treated as an external crate. We have to initialize the default then mutate it.